site/content/introduction/03-Features.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Features"
+weight: 30
+---
+
+### Feature list [WIP]
+
+The features that are currently implemented are:
+
+- Create cluster
+- Get cluster
+- ...
```

Comment:
I'll add a couple points here too.

Reply:
Wait, maybe it's not ideal to pursue this format?
The same things that I would consider features are already headings under 'USAGE' :

- Creating and managing clusters
- Managing nodegroups
- Cluster Upgrades
- Autoscaling
- Custom AMI support
- VPC Networking
- GPU Support
- Spot instances
- Supported IAM Add-on Policies
- Manage IAM users and roles

We could change that title from 'USAGE' to 'FEATURES AND USAGE'?
What do you think @martina-if ?

Reply:
Maybe  we could add a short list with the main features to the introduction? As an explanation of what eksctl can do. Then keep the usage as the proper list of features. About the name of the section I have no strong opinions. Both sound good to me.

Reply:
Ok!